VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -975,15 +978,21 @@ private boolean isScreenLandscape() {
         @Override
         public void run() {
             if (isResumed() && !isRemoving()) {
-                hideProgress();
-                if(player!=null) {
+                if (player != null) {
                     player.unfreeze();
+                    hideProgress();
                     if (player.isPlaying()) {
-                        hideProgress();
                         updateController("player unfreezed");
                     }
 
-                    if (pauseDueToDialog){
+                    if (IS_AUTOPLAY_ENABLED && isPrepared && !isPaused) {
+                        player.start();
+                    } else {
+                        updateController("player paused");
+                    }
```

Comment:
I think starting the player should not need to be performed from this delayed unfreeze callback; you should be able to start it directly from `onResume()`.